packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md

```diff
@@ -6,6 +6,20 @@ title: Okta Identity Engine API Products Release Notes 2021
 
 ## August
 
+### Weekly Release 2021.08.2
+
+| Change                                                                     | Expected in Preview Orgs |
+|----------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2021.08.2](#bug-fixed-in-2021-08-2)                          | August 18, 2021          |
+
+#### Bugs fixed in 2021.08.2
+
+- In the [Device Authorization flow](/docs/guides/device-authorization-grant/main/), the URI link that was used in a QR Code was missing if the org wasn't on the Identity Engine. (OKTA-413425)
+
+- When using the `/token` endpoint, OAuth 2.0 refreshed the [access and ID tokens](/docs/guides/refresh-tokens/overview/) for all application users, which included deactivated users instead of just active users. (OKTA-417991)
+
+- In cases where a suspended user, deactivated or unknown user, or a valid user with the wrong password tried to sign in, an HTTP 400 response code was returned instead of an HTTP 401 response code. (OKTA-418023)
```

Comment:
```suggestion
- In cases where a suspended user, deactivated or unknown user, or a valid user with the wrong password tried to sign in, an HTTP 400 response code was returned instead of an HTTP 401 response code. (OKTA-418923)
```